package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hubot-ingress",
   "description": "Ingress commands for Hubot",
-  "version": "0.19.0",
+  "version": "0.19.2",
```

Comment:
Revert this change, please. Version number changes are handled by the package maintainer(s), not contributors. 👍